packages/grid/_modules_/grid/hooks/features/keyboard/useGridKeyboard.ts

```diff
@@ -112,7 +107,7 @@ export const useGridKeyboard = (apiRef: GridApiRef): void => {
       }
 
       if (isNavigationKey(event.key) && !isSpaceKey(event.key) && !event.shiftKey) {
-        apiRef.current.publishEvent(GRID_COLUMN_HEADER_NAVIGATION_KEY_DOWN, params, event);
+        apiRef.current.publishEvent(GridEvents.cellNavigationKeyDown, params, event);
```

Comment:
It should have been:

```suggestion
        apiRef.current.publishEvent(GridEvents.columnHeaderNavigationKeyDown, params, event);
```

The keyboard navigation is no longer working on the header https://codesandbox.io/s/material-demo-forked-git2x